crates/rpc/src/handlers/validator.rs

```diff
@@ -37,21 +34,32 @@ impl ValidatorData {
     }
 }
 
+#[get("/beacon/states/{state_id}/validator/{validator_id}")]
 pub async fn get_validator_from_state(
-    state_id: ID,
-    validator_id: ValidatorID,
-    db: ReamDB,
-) -> Result<impl Reply, Rejection> {
-    let state = get_state_from_id(state_id, &db).await?;
+    db: web::Data<ReamDB>,
+    param: web::Path<(ID, ValidatorID)>,
+    web::Json(validator): web::Json<Validator>,
```

Comment:
```suggestion
    validator: web::Json<Validator>,
```

Could you remove all usage of this